Packs/MicrosoftGraphDeviceManagement/Integrations/MicrosoftGraphDeviceManagement/MicrosoftGraphDeviceManagement.py

```diff
@@ -0,0 +1,428 @@
+            'BootDebugging': device_health_attestation_state.get('bootDebugging'),
+            'OperatingSystemKernelDebugging': device_health_attestation_state.get('operatingSystemKernelDebugging'),
+            'CodeIntegrity': device_health_attestation_state.get('codeIntegrity'),
+            'TestSigning': device_health_attestation_state.get('testSigning'),
+            'SafeMode': device_health_attestation_state.get('safeMode'),
+            'WindowsPE': device_health_attestation_state.get('windowsPE'),
+            'EarlyLaunchAntiMalwareDriverProtection':
+                device_health_attestation_state.get('earlyLaunchAntiMalwareDriverProtection'),
+            'VirtualSecureMode': device_health_attestation_state.get('virtualSecureMode'),
+            'PCRHashAlgorithm': device_health_attestation_state.get('pcrHashAlgorithm'),
+            'BootAppSecurityVersion': device_health_attestation_state.get('bootAppSecurityVersion'),
+            'BootManagerSecurityVersion': device_health_attestation_state.get('bootManagerSecurityVersion'),
+            'TPMVersion': device_health_attestation_state.get('tpmVersion'),
+            'PCR0': device_health_attestation_state.get('pcr0'),
+            'SecureBootConfigurationPolicyFingerPrint':
+                device_health_attestation_state.get('secureBootConfigurationPolicyFingerPrint'),
+            'CodeIntegrityPolicy': device_health_attestation_state.get('codeIntegrityPolicy'),
+            'BootRevisionListInfo': device_health_attestation_state.get('bootRevisionListInfo'),
+            'OperatingSystemRevListInfo': device_health_attestation_state.get('operatingSystemRevListInfo'),
+            'HealthStatusMismatchInfo': device_health_attestation_state.get('healthStatusMismatchInfo'),
+            'HealthAttestationSupportedStatus': device_health_attestation_state.get('healthAttestationSupportedStatus')
+        }),
+        'SubscriberCarrier': raw_device.get('subscriberCarrier'),
+        'MEID': raw_device.get('meid'),
+        'TotalStorageSpaceInBytes': raw_device.get('totalStorageSpaceInBytes'),
+        'FreeStorageSpaceInBytes': raw_device.get('freeStorageSpaceInBytes'),
+        'ManagedDeviceName': raw_device.get('managedDeviceName'),
+        'PartnerReportedThreatState': raw_device.get('partnerReportedThreatState')
+    })
+
+
+def build_device_human_readable(device: dict) -> dict:
+    """
+    Builds a device human readable object
+    :param device: The raw device object
+    :return: The device human readable object
+    """
+    device_human_readable: dict = dict()
+    for header in HEADERS['raw_device']:
+        index: int = HEADERS['raw_device'].index(header)
+        device_human_readable[HEADERS['device'][index]] = device.get(header)
+    return assign_params(**device_human_readable)
+
+
+''' COMMANDS '''
+
+
+def list_managed_devices_command(client: MsGraphClient, args: dict) -> None:
+    list_raw_devices, raw_response = client.list_managed_devices(args)
```

Comment:
As we discussed, extract the relevant arguments from `args` and pass them explicitly to the client function.
Go over all commands and do the same.